### ISSUE TITLE
Improve photogame

### ISSUE THREAD

**Issue by FamTroder** [FamTroder/FamMel#90]
It's really boring right now.
Options include:
- Use the same variables used during sex to keep track of how naked she is.
- Instruct which pose she uses.

**Comment by M0nkey5ee** [comment]
Love the game and agree, would love to (greatly if you like) enhance it! Looking forward to your ideas / the ones of the community maybe? Can come up with smt. on my own of course as well if needed, just right now nothing rings a bell.

**Comment by Forget-About-Me2** [comment]
Yeah, I'm not really sure what to do with it. The photo game currently is really just you asking her to pose and she does it until she loses control or you let her go to the toilet. I guess it kinda misses a win condition, as it's not really a game. Could be you offering a favour if she manages to do a certain amount of posses or something else, not sure.

**Comment by M0nkey5ee** [comment]
Is there anything you want to deepen your coding-skills anyway in? If so, could a new feature be used as practise for this?
If not, what about her posing with one of these chairs that I don't understand the normal use-case of but make sense here? (https://www.pinterest.de/pin/my-bloomin-chair-vintage-solid-wood-chair-painted-cream-plant-pot-hole-in-seat-use-indoors-or-outside-in-the-g--174725660514034778/) - Maybe she is challenged to pose / sit on it for x minutes without losing y amounts of drops or more?

**Comment by Forget-About-Me2** [comment]
Hmm no not really.
The idea of the chair could be fun, but I can't really imagine a practical way to implement it. 
Honestly might be better to focus on some other features and new locations instead of spicing up the club, unless a brilliant idea for it comes along.
